src/Joomlatools/Console/Command/Site/Download.php

```diff
@@ -200,7 +221,7 @@ protected function _getTarball(OutputInterface $output)
         // We can be certain that the joomla-cms repository is a public GitHub repository
         // so we can download the files straight over HTTP.
         // We have no clue about anything else, so we clone those locally and fall back on git-archive
-        if ($repository == 'https://github.com/joomla/joomla-cms.git')
+        if (!$this->clone || $repository == 'https://github.com/joomla/joomla-cms.git')
```

Comment:
@msaladna This option can't work for another mirror because we call the `_downloadJoomlaCMS()` method inside the `if` block, which is currently using the hardcoded default Joomla URL. It doesn't take  See: https://github.com/apisnetworks/joomlatools-console/blob/6ef1baadd26f24863539c7791e013a2c49b030c3/src/Joomlatools/Console/Command/Site/Download.php#L263-L276 

Are there any changes missing here?

Reply:
`_downloadJoomlaCMS()` now negotiates the correct repo to use. Either github or mirror if --repo=<location> used.